modules/nf-core/plink/genome/main.nf

```diff
@@ -0,0 +1,39 @@
+process PLINK_GENOME {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/plink:1.90b6.21--h031d066_5':
+        'biocontainers/plink:1.90b6.21--h031d066_5' }"
+
+    input:
+    tuple val(meta), path(bed), path(bim), path(fam)
+
+    output:
+    tuple val(meta), path("*.genome")   , emit: genome
```

Comment:
Move the emits closer, the comma should be right after the longest text

Reply:
Thank you, I've fixed this